charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -183,44 +213,42 @@ spec:
         {{- if .Values.repoServer.volumeMounts }}
           {{- toYaml .Values.repoServer.volumeMounts | nindent 8 }}
         {{- end }}
-        - mountPath: /app/config/gpg/source
-          name: gpg-keys
-        - mountPath: /app/config/gpg/keys
-          name: gpg-keyring
-        {{- if .Values.configs.knownHosts }}
```

Comment:
Not an optional volume. All optional volumes are marked as such via `optional: true`